metis-common-security/src/main/java/eu/europeana/metis/security/test/JwtUtils.java

```diff
@@ -0,0 +1,83 @@
+package eu.europeana.metis.security.test;
+
+import eu.europeana.metis.security.AccountRole;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.security.oauth2.jwt.Jwt;
+
+/**
+ * This module is only intended for use in test code. It provides test data factories and shared test utilities.
+ */
+
+public class JwtUtils {
+
+  public static final String BEARER = "Bearer ";
```

Comment:
only this is not used, should a test at least include it? or this is used somewhere else?